CMakeLists.txt

```diff
@@ -454,12 +456,7 @@ message(STATUS "CUDA Compiler is" ${CMAKE_CUDA_COMPILER})
 message(STATUS "Compiler ID is " ${CMAKE_CUDA_COMPILER_ID})
 
 # CUDA Wrapper for finding libs etc
-if(CMAKE_VERSION VERSION_LESS 3.17)
-  find_package(CUDAWrapper)
-else()
-  # for cmake 3.17+ we rely on
-  find_package(CUDAToolkit)
-endif()
+find_package(CUDAToolkit)
```

Comment:
Since we require minimum 3.17 here now, I could remove this if() to simplify code